doc/source/examples/unresolved-cfd-dem/boycott-effect/boycott-effect.rst

```diff
@@ -0,0 +1,372 @@
+
+
+The additional sections for the CFD-DEM simulations are the void fraction subsection and the CFD-DEM subsection. These subsections are described in detail in the `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ .
+
+Void fraction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We choose to use the quadrature centered method (QCM) to calculate the void fraction. This method does not require smoothing the void fraction as it is space and time continuous. 
+ 
+.. code-block:: text
+
+    subsection void fraction
+   	set mode = qcm
+   	set read dem = true
+   	set qcm sphere equal cell volume = false
+   	set dem file name = dem
+   	set l2 smoothing factor = 0
+   	set bound void fraction = false
+    end
+
+CFD-DEM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We also enable grad_div stabilization in order to improve local mass conservation. The void fraction time derivative is disabled as the two phases have close densities and as such the time variation of the void fraction is unstable. 
+
+.. note:: 
+    For certain simulations, this parameter should be disabled to improve stability of the solver.
+
+.. code-block:: text
+
+    subsection cfd-dem
+   	set grad div = true
+   	set void fraction time derivative = false
+   	set drag force = true
+   	set buoyancy force = true
+   	set shear force = true
+   	set pressure force = true
+   	set drag model = difelice
+   	set post processing = true
+   	set coupling frequency = 250
+   	set implicit stabilization = false
+   	set grad-div length scale = 0.005
+   	set vans model = modelA
+    end
+    
+We determine the drag model to be used for the calculation of particle-fluid forces. We enable buoyancy, drag, shear and pressure forces. For drag, we use the Difelice model to determine the momentum transfer exchange coefficient. The VANS model we are solving is model A. Other possible option is model B.
+
+Finally, the linear and non-linear solver controls are defined.
+
+Non-linear solver control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection non-linear solver
+        set solver = inexact_newton
+   	set tolerance               = 1e-8
+  	set max iterations          = 10
+  	set verbosity               = verbose
+  	set matrix tolerance        = 0.75
+    end
+
+We use the inexact_newton solver as to avoid the reconstruction of the system matrix at each Newton iteration. For more information about the non-linear solver, please refere to the `Non Linear Solver Section <../../../parameters/cfd/non-linear_solver_control.html>`_
```

Comment:
```suggestion
We use the inexact newton solver as to avoid the reconstruction of the system matrix at each Newton iteration. For more information about the non-linear solver, please refer to the `Non Linear Solver Section <../../../parameters/cfd/non-linear_solver_control.html>`_
```